util/fibers/uring_file.h

```diff
@@ -59,14 +63,24 @@ class LinuxFile {
 
   std::error_code ReadFixed(io::MutableBytes dest, off_t offset, unsigned buf_index);
 
+  // int - io result.
   using AsyncCb = std::function<void(int)>;
 
   // Async versions of Read
   void ReadFixedAsync(io::MutableBytes dest, off_t offset, unsigned buf_index, AsyncCb cb);
   void ReadAsync(io::MutableBytes dest, off_t offset, AsyncCb cb);
+  void WriteFixedAsync(io::Bytes src, off_t offset, unsigned buf_index, AsyncCb cb);
+  void WriteAsync(io::Bytes src, off_t offset, AsyncCb cb);
```

Comment:
With those changes, can't we then just replace our IoMgr from DF with a plain LinuxFile?

Reply:
Btw, what do you think about overloads accepting UringBuf right here?

Reply:
> With those changes, can't we then just replace our IoMgr from DF with a plain LinuxFile?

That's the plan.

Reply:
> Btw, what do you think about overloads accepting UringBuf right here?

Why is it better?

Reply:
> Why is it better?

Just more convenient, we use fixed reads only with uring buf